exchanges/okx/okx_wrapper.go

```diff
@@ -1278,8 +1252,8 @@ func (ok *Okx) CancelOrder(ctx context.Context, ord *order.Cancel) error {
 		if err != nil {
 			return err
 		}
-		if response.StatusCode != "0" {
-			return fmt.Errorf("sCode: %s sMessage: %s", response.StatusCode, response.StatusMessage)
+		if response.StatusCode != 0 {
```

Comment:
```suggestion
		return getStatusError(response.StatusCode, response.StatusMessage)
```

Since the func does this check anyway and will make a neglible impact in terms of perf since the compiler will inline getStatusError 

![image](https://github.com/user-attachments/assets/600a37b4-5e46-428f-98e8-b686184ac0b5)

Reply:
🔥 [1333b42](https://github.com/thrasher-corp/gocryptotrader/pull/1851/commits/1333b42b7c47a5e8e03dc1ae59f4f28c967d6d9a)